docs/pages/_app.js

```diff
@@ -195,36 +196,6 @@ Tip: you can access the documentation \`theme\` object directly in the console.
   );
 }
 
-// DO NOT REPLACE THIS FUNCTION WITH THE ONE FROM THE MONOREPO
```

Comment:
Just in case, why was this comment here and did you check that we could replace it now ?

Reply:
> did you check that we could replace it now

These are what I checked:
- the e2e-test make sure the page does not crash
- the footer of the docs works as before (can link to the next page)

Reply:
I suggest you try the migration steps I mentioned in the PR description.